cmd/juju/machine/upgradeseries.go

```diff
@@ -34,6 +36,16 @@ of machine %q will also be upgraded. These units include:
 
 Continue [y/N]?`[1:]
 
+const UpgradeSeriesPrepareFinishedMessage = `
+Juju is now ready for the series to be updated.
+Perform any manual steps required along with "do-release upgrade".
```

Comment:
Can you tweak to: do-release-upgrade
That's the CLI command to upgrade.

Reply:
Yup can do

Reply:
typo fixed